src/components/task-step/exercise.cjsx

```diff
@@ -0,0 +1,46 @@
+React = require 'react'
+
+{TaskStepActions, TaskStepStore} = require '../../flux/task-step'
+{TaskStore} = require '../../flux/task'
+{StepPanel} = require '../../helpers/policies'
+
+{Exercise} = require 'openstax-react-components'
+StepFooter = require './step-footer'
+
+module.exports = React.createClass
+  displayName: 'ExerciseShell'
+  propTypes:
+    id: React.PropTypes.string.isRequired
+    taskId: React.PropTypes.string.isRequired
+
+  render: ->
+    {id, taskId} = @props
+    step = TaskStepStore.get(id)
+    task = TaskStore.get(taskId)
+
+    stepProps = {}
```

Comment:
Why pull the props out instead of explicitly putting them on the `<Exercise>`? It looks like the props are always set